src/test/java/core/basesyntax/dao/FruitStoreDaoTest.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitStoreDaoTest {
+    private static final int QUANTITY = 5;
+    private static FruitStoreDao fruitStoreDao;
+    private static Map<Fruit, Integer> expectedMap;
+    private static Fruit apple;
+
+    @BeforeClass
+    public static void beforeAll() {
+        fruitStoreDao = new FruitStoreDaoImpl();
+        expectedMap = new HashMap<>();
+        apple = new Fruit("apple");
+        Storage.getFruits().put(apple, QUANTITY);
+    }
+
+    @Test
+    public void addFruit_equalsMap_returnsTrue() {
```

Comment:
Same with equalsMap it is not descriptive. As example whenAddApple65_dBContains

Reply:
renamed